Add module to import OrdinaryDiffEq

sibyjackgrove opened the pull request:
This change will allow users of `diffeqpy `to use `OrdinaryDiffEq `which is a lighter weight package compared to `DifferentialEquations`. It will with decreasing loading time and sub-process memory consumption.

Usage: `from diffeqpy import ode`

ChrisRackauckas commented:
This is a good idea. Add it to the README?

sibyjackgrove commented:
> This is a good idea. Add it to the README?

Sure! Would you like me to do it or will you be doing it?

ChrisRackauckas commented:
If you could add a section on this that would be great. Also, it should get a test. Then I think it's ready to merge.

sibyjackgrove commented:
> If you could add a section on this that would be great. Also, it should get a test. Then I think it's ready to merge.

@ChrisRackauckas Sure. I have updated README and added a unit test.

ChrisRackauckas commented:
Great thanks! I'll need to update the version on PyPI when I get a chance, but for now it can be used locally.
